helm/druid/Chart.yaml

```diff
@@ -1,39 +1,10 @@
-# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
We shouldn't remove these license headers, we can remove them when we release Helm chart by [helm-chart-releaser](https://github.com/marketplace/actions/helm-chart-releaser).